main1.css

```diff
@@ -0,0 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap');
+
+html, body{
+  background: #A0D9CB;
+  height: 100vh;
+  font-family: 'Roboto', sans-serif;
+}
+
+.header-img {
+  width:200px;
+}
+
+.item {
+  background: #ff9494;
+  padding: 16px 12px 8px;

```

Comment:
Šo var panākt ar bootstrap.
Varētu krāsu iznest kā klasi un tad pielikt tikai to.